code/modules/mob/living/simple_animal/abnormality/teth/faelantern.dm

```diff
@@ -236,7 +236,7 @@
 			if(WEST)
 				R.pixel_x -= 16
 		for(var/mob/living/L in T)
-			L.apply_damage(root_damage, RED_DAMAGE, null, spread_damage = TRUE)
+			L.apply_damage(root_damage, RED_DAMAGE, spread_damage = TRUE)
```

Comment:
Is this line meant to use apply_damage and not the new deal_damage proc?

Reply:
for documentation = yesn't
deal_damage automatically applies armor reduction, so putting this into that proc raw would result in the armor activelly blocking it, whilst its not the intention
But it is supposed to be changed to BRUTE damage as we use it for penetrating red effects